src/D2L.Bmx/Utilities.cs

```diff
@@ -0,0 +1,9 @@
+
+namespace D2L.Bmx;
+internal class Utilities {
```

Comment:
we can do better here... something named "utilities" can often end up as a dumping ground.
maybe `BmxPaths`?

Reply:
Sure. That sounds fine